bin/commander.js

```diff
@@ -31,6 +31,16 @@ const success = chalk.bold.green;
 
 // Utils
 
+/**
+ * Small utility used to read an uncached version of a JSON file
+ *
+ * @param {string} fileName
+ */
+function readJSONFile( fileName ) {
+	const data = fs.readFileSync( fileName, 'utf8' );
```

Comment:
There is also:

https://www.npmjs.com/package/read-pkg

Not sure if it’s worth the hassle though 😅

Reply:
We also read the lock file, and it's simple enough :)

Reply:
Correct, let's move forward with this PR 👍